src/Collect/Support/Collection.php

```diff
@@ -34,7 +34,7 @@ class Collection implements ArrayAccess, Arrayable, Countable, IteratorAggregate
      */
     protected static $proxies = [
         'average', 'avg', 'contains', 'each', 'every', 'filter', 'first', 'flatMap',
-        'keyBy', 'map', 'partition', 'reject', 'sortBy', 'sortByDesc', 'sum',
+        'keyBy', 'map', 'partition', 'reject', 'sortBy', 'sortByDesc', 'sum', 'unique',
```

Comment:
Is this something we missed when we upgraded to the latest? @besologic Could you check?

Reply:
This is Laravel [5.5.33](https://github.com/laravel/framework/blob/v5.5.33/src/Illuminate/Support/Collection.php#L37)

Reply:
ahh, we're only up to `.32`